src/lib/openapi/spec/environment-create-schema.ts

```diff
@@ -0,0 +1,27 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const environmentCreateSchema = {
```

Comment:
Yeah, we can either (1) write the schemas and derive types from them, or (2) write types and generate the schema.

We went with the first approach. I like it since it doesn't require a code gen step. But definitely pros and cons.

Reply:
Interesting. We discussed doing something like that when we first started integrating OpenAPI. I was supposed to look into it, but the solution I found wasn't appropriate for the Proxy, so I never got around to actually implementing it. That is closer to what I'm used to from other programming languages though, and with how finicky the type generation _can_ be, I'm not against it.